libmscore/textbase.cpp

```diff
@@ -936,11 +936,35 @@ void TextBlock::layout(TextBase* t)
                         break;
                   }
             }
+
       if (_fragments.empty()) {
             QFontMetricsF fm = t->fontMetrics();
             _bbox.setRect(0.0, -fm.ascent(), 1.0, fm.descent());
             _lineSpacing = fm.lineSpacing();
             }
+      else if (_fragments.size() == 1 && _fragments.at(0).text == "") {
```

Comment:
No need to compare against a literal C-style string. Just call `isEmpty()`:

    else if ((_fragments.size() == 1) && _fragments.at(0).text.isEmpty()) {

(I also added parentheses the first expression to conform to the coding guidelines.)